src/analogPins.cpp

```diff
@@ -0,0 +1,33 @@
+#include "analogPins.h"
+
+// Ticker to poll input readings at a fixed rate
+STM32TimerInterrupt dataUpdater(TIM2);
+
```

Comment:
You need to initialize the volatile floats here since the ones in the `.h` file were declared as `extern`